src/form/FileUpload/FileUploadStyles.ts

```diff
@@ -16,9 +16,14 @@ export const FileUploadStyle = styled.div`
 	cursor: pointer;
 `;
 
+export const FileUploadPlaceholderWrapper = styled.div`
+	display: flex;
+	gap: ${themeSpacing(2)};
+`;
+
 export const FileUploadPlaceholderTextCss = css`
 	font-weight: 400;
-	font-size: 24px;
+	font-size: 18px;
```

Comment:
Font size but not the line-height? This already conflicts the DS. I would prefer it if this was inherited from the body or set by variables.

Reply:
Ok, noted. We will make some better decisions about font styling and how to apply them soon